tests/test_number_spelling.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""Testing the spelling of numbers
+
+:copyright: (c) 2013 by Szabolcs Blága
+:license: BSD, see LICENSE in babel for more details.
+"""
+from __future__ import unicode_literals
+
+import pytest
+
+from babel import numbers
+from babel import number_spelling as numspell
+
+
+def test_context_framework():
+    cm = numspell.ContextMaker()
+    c1 = cm(2222.22222)
+
+    for side in c1:
+        assert isinstance(side, numspell.SideContext)
+        for group in side:
+            assert isinstance(group, numspell.GroupContext)
+            for digit in group:
+                assert isinstance(digit, numspell.DigitContext)
+                assert digit.value == 2
+
+
+def test_en_GB_cardinal():
+    assert numbers.spell_number(0, locale='en_GB') == "zero"
+    assert numbers.spell_number(1, locale='en_GB') == "one"
+    assert numbers.spell_number(2, locale='en_GB') == "two"
+    assert numbers.spell_number(3, locale='en_GB') == "three"
+    assert numbers.spell_number('-0', locale='en_GB') == "minus zero"
+    assert numbers.spell_number(123.25, locale='en_GB') == "one hundred and twenty-three point twenty-five hundredths"
+    assert numbers.spell_number(-12, locale='en_GB') == "minus twelve"
+    assert numbers.spell_number(23457829, locale='en_GB') == "twenty-three million four hundred and fifty-seven thousand eight hundred and twenty-nine"
+    assert numbers.spell_number(1950, locale='en_GB') == "one thousand nine hundred and fifty"
+    assert numbers.spell_number(2001, locale='en_GB') == "two thousand one"
+    assert numbers.spell_number('1999.238', locale='en_GB') == "one thousand nine hundred and ninety-nine point two hundred and thirty-eight thousandths"
+    assert numbers.spell_number(-.199923862, locale='en_GB') == "rounded to minus zero point two tenths"
+    assert numbers.spell_number(-.1, locale='en_GB') == "minus zero point one tenth" # float to string conversion preserves precision
+
```

Comment:
Insert a new line here. Take a look at the Pocoo Styleguide: http://www.pocoo.org/internal/styleguide/